File: python/get_floor_and_entrance.go
package main

import "fmt"

func getFloorAndEntrance(flatNumber int) (int, int) {
	// Calculate the floor number
	floor := (flatNumber - 1) / 20 + 1

	// Calculate the entrance number
	entrance := ((flatNumber - 1) % 20) / 4 + 1

	return floor, entrance
}

func main() {
	floor, entrance := getFloorAndEntrance(21)
    fmt.Printf("Floor: %d, Entrance: %d\n", floor, entrance)
}

File: python/get_floor_and_entrance.py
from typing import Tuple

def get_floor_and_entrance(flat_number: int) -> Tuple[int, int]:
    entrance = (flat_number - 1) // 20 + 1
    floor = (flat_number - 1) % 20 // 4 + 1

    return entrance, floor

print(get_floor_and_entrance(25))
